### PR TITLE
Hw25

### PR DESCRIPTION
**Pull request by iibrahimsv**
Homework Checking Team,

Homework 25 is ready for review. Please advise. 👍🏽 

Best Wishes

-Abdi Ibrahim

### CODE REVIEW

**Comment by Berichicko** [comment]
Hi, Ibrahim

Please push the` class BaseTest` for checking in Hw25

Thanks!

File: src/test/java/LoginTests.java [hunk 1]
```diff
@@ -14,12 +14,35 @@
...
+    public void LoginEmptyPasswordTest () {
+        LoginPage loginPage = new LoginPage(driver);
+
+        loginPage.provideEmail("demo@class.com");
+        loginPage.providePassword("");
+        loginPage.clickSubmitBtn();
+        Assert.assertTrue(loginPage.isPageOpened());
+
+    }
+
```
Comment (Berichicko) [inline src/test/java/LoginTests.java:39]: I fixed it:

image

image

Here is the solution:

image

**Comment by Berichicko** [comment]
Hello

Homework  25  is cheeked! 

I left comments on your code!


**Excellent!**  🔥


**Your grade: 95**  ✊


💖Best wishes, the team HW (Alex)!💖

File: src/test/java/LoginTests.java (hunk 1 above)
Comment (iibrahimsv) [reply src/test/java/LoginTests.java:39]: I've tried all locators and sometimes it works and sometimes it doesn't.

**Comment by iibrahimsv** [comment]
Thank you for the feedback @Berichicko ! Will review and make changes. 👍🏽

**Comment by Berichicko** [comment]
Thank you very much!

**Pull request closed without merging**